docs/API.md

```diff
@@ -0,0 +1,175 @@
+# Peerpad core API
+
+## Install
+
+```bash
+$ npm install peerpad-core --save
+```
+
+## Import
+
+```js
+import Peerpad from 'peerpad-core'
+```
+
+## `Peerpad(options)`
+
+Creates a Peerpad
+
+```js
+const options = {
+  name: 'name of the pad',
+  readKey: 'gobelegook',
+  writeKey: 'moregobelegook'
+}
+
+const peerpad = Peerpad(options)
+```
+
+## `options`:
+
+* `name`: string that uniquely identifies this
+* `readKey`: b58-encoded string or buffer that contains the read key
+* `writeKey`: b58-encoded string or buffer that contains the write key (optional)
```

Comment:
Needs signing key (probably it is using the one from IPFS, but ideally, that should be an explicit option rather than just assuming that key is always present in an IPFS node).